### ISSUE TITLE
rem_vertex! breaks when indexing properties are in use

### ISSUE THREAD

**Issue by sbromberger** [sbromberger/MetaGraphs.jl#78]
```
julia> g
{2902, 13599} directed Int64 metagraph with Float64 weights defined by :weight (default weight 1.0)
julia> rem_vertex!(g, 4)
ERROR: The following properties are indexing_props and cannot be updated: Set(Symbol[:name])
```

**Comment by Affie** [comment]
It is fixed already in master, see #72 
Can someone perhaps tag a new release?

**Comment by sbromberger** [comment]
Fixed in #73.
